sdk/eventhub/azure-eventhub/azure/eventhub/__init__.py

```diff
@@ -33,5 +34,6 @@
     "LoadBalancingStrategy",
     "PartitionContext",
     "parse_connection_string",
-    "EventHubConnectionStringProperties"
+    "EventHubConnectionStringProperties",
+    "RetryMode"
```

Comment:
There're API changes, remember to get @annatisch to take a look at the API view before CC day :)